src/achievement.h

```diff
@@ -1,3 +1,4 @@
+#pragma once
 #ifndef CATA_SRC_ACHIEVEMENT_H
 #define CATA_SRC_ACHIEVEMENT_H
```

Comment:
```suggestion
#define CATA_SRC_ACHIEVEMENT_H,
```
Resetting test suite